lib/bleno.js

```diff
@@ -51,9 +52,21 @@ Bleno.prototype.Descriptor = Descriptor;
 Bleno.prototype.onStateChange = function(state) {
   debug('stateChange ' + state);
 
+  var self = this;
+
   this.state = state;
 
-  this.emit('stateChange', state);
+  function emit(OSPlatform) {
+    self.platform = OSPlatform;
+    self.emit('stateChange', state);
+  }
+
+  if(this.platform){
+    this.emit('stateChange', state);
+  }else{
+    self._bindings.once('OSPlatform', emit);
+    self._bindings.platform();
```

Comment:
@jacobrosenthal I'm wondering if the ``this.emit('OSPlatform', os.platform());`` can be moved into the binding ``init`` method, for example:
* https://github.com/sandeepmistry/bleno/blob/master/lib/hci-socket/bindings.js#L71
* https://github.com/sandeepmistry/bleno/blob/master/lib/mac/bindings.js#L66

such that the platform is always emitted before the stateChange.